pandas/io/gbq.py

```diff
@@ -14,21 +19,21 @@ def _try_import():
 
 
 def read_gbq(
-    query,
-    project_id=None,
-    index_col=None,
-    col_order=None,
-    reauth=False,
-    auth_local_webserver=False,
-    dialect=None,
-    location=None,
-    configuration=None,
+    query: str,
+    project_id: Optional[str] = None,
+    index_col: Optional[str] = None,
+    col_order: Optional[List[str]] = None,
```

Comment:
Does this explicitly need to be a List or does Sequence work as well?

Reply:
Tried the other basic sequence types and they throw. Since it's used to retrieve columns from a DataFrame. Like `df[col_order]`